x-pack/plugins/siem/server/graphql/kpi_hosts/schema.gql.ts

```diff
@@ -12,23 +12,23 @@ export const kpiHostsSchema = gql`
     doc_count: Float
   }
 
-  type HistogramData {
+  type KpiHostHistogramData {
     key: Float
     key_as_string: String
     count: Count
   }
 
   type KpiHostsData {
     hosts: Float
-    hostsHistogram: [HistogramData]
+    hostsHistogram: [KpiHostHistogramData]
```

Comment:
FYI -> if you do [KpiHostHistorgramData!], it will change the type from `(KpiHostHistogramData | null)[] | null;}` to `(KpiHostHistogramData)[] | null;}` and I think that can simplify some of your logic.